README.md

```diff
@@ -58,18 +58,17 @@ You can either
 - server-communication (Rest APIs, etc.)
 - [documentation](https://docs.fritz2.dev) (work in progress)
 
-## What will come next?
+## What is next for fritz2?
 
 - performance and memory optimizations
```

Comment:
We should remove this list here and link to our GitHub issues...

Reply:
Go ahead :-)

Reply:
just remove this part